src/adapter.js

```diff
@@ -59,6 +60,84 @@ function useResourceGroups(api) {
   return tags.length > 0;
 }
 
+// Look up a position in the original source based on a JSON path, for
+// example 'paths./test.get.responses.200'
+function getPosition(ast, path) {
```

Comment:
The input JSON path should probably support some kind of escaping mechanism. Maybe just percent-encode the periods and square brackets? Ideas?